### DISCUSSION TITLE
What does L 1.** signify when POWERLEVEL9K_RIGHT_PROMPT_ELEMENTS have been modified?

### DISCUSSION THREAD

**Discussion by harr1424** [romkatv/powerlevel10k#2740] [General]
I've modified my POWERLEVEL9K_RIGHT_PROMPT_ELEMENTS to display some basic information such as hdd usage, available RAM, and IP/throughput. Any time I uncomment these components in order for them to be displayed, the very first (left-most) component shown ion the POWERLEVEL9K_RIGHT_PROMPT_ELEMENTS is the character 'L' with a floating point number between 1.5 and 2.0.  What does this component relate to? 

<img width="594" alt="image" src="https://github.com/user-attachments/assets/bffecea5-90c6-493a-b38a-c764bdf097ad">

**Comment by romkatv** [comment]
That's `load`: https://github.com/romkatv/powerlevel10k/blob/2b7da93df04acd04d84f5de827e5b14077839a4b/config/p10k-lean.zsh#L93

You could figure this out by enabling prompt segments one by one rather than all at once.

Also see https://github.com/romkatv/powerlevel10k/blob/2b7da93df04acd04d84f5de827e5b14077839a4b/config/p10k-lean.zsh#L804-L812

**Comment by harr1424** [comment]
Thanks, Roman! Very helpful.
